src/index.ts

```diff
@@ -1,3 +1,12 @@
 export * from './auth'
 export * from './session'
 export * from './common'
+
+declare global {
```

Comment:
Out of curiosity asking why you had to do this? Compiler complaining?

Reply:
yeah we have our own typings.d.ts but when building the project it doesn't get included into the dist folder. Also in package.json the typings is tied too the index.d.ts (built file), this was the only way I could get the req object to be extended. Linter then complained so have had to do this :/